### DISCUSSION TITLE
Snippet not rendering after build

### DISCUSSION THREAD

**Discussion by DarksideVT** [facelessuser/pymdown-extensions#1903] [Q&A] [answered]
I've been running a dev env of my site and using the snippet renders when I use `mkdocs serve`. After I `mkdocs build` and host it on an nginx server, the entire website works other than the snippet. Am I not able to use snippets when I build then host on an nginx site? You can take a look at my website to see what I mean. https://ed-tech.tech/Docker/Python_Dev_containerization/ About halfway down the page there are 2 codeblocks that have nothing in them. That's where the snippet should be going.

**Comment by facelessuser** [comment, marked as answer]
I know nothing about your system nor how you are using Snippets.

Snippets is run the same way in both `mkdocs serve` and `mkdocs build`. When the page is being rendered, the snippet is first injected, then the page is rendered. The only difference between `serve` and `build` is that `serve` runs a local dev sever for live viewing. They should be built in the exact same way. As a matter of fact, Snippets is completely agnostic about what MkDocs is doing as it is not specifically tied to MkDocs in any way. It is a Python Markdown extension, not a MkDocs plugin, so it doesn't care whether you are building or serving your docs.

I would need far more information about your project and the environment under which you are building your project than I currently have. Better yet, I would advice constructed a minimal, reproducible example. You post a link to a zip or a repo containing this minimal example if you like.

**Comment by DarksideVT** [comment]
Here is the markdown portion that corresponds to this. 
````
```py title="Dockerfile" linenums="1"
--8<-- "docs/Docker/python-docker/dockerfile"
```
```bash title="init.bash" linenums="1"
--8<-- "docs/Docker/python-docker/init.bash"
```
````

**Reply by facelessuser** [reply]
Again, there should be no issues assuming when built or served that Snippets can actually find the files at the locations you specify. I do not have enough information to know whether this is true or not.

**Reply by DarksideVT** [reply]
I think this is on me. Looks like the way that I'm building it, it doesn't pull that repo down at all... that's on me.

**Reply by facelessuser** [reply]
Okay, that is kind of what I figured it most likely to be, an "environment issue". Such questions require information that I am just not privy to.
